modules/cnv_install_1_4.adoc

```diff
@@ -0,0 +1,64 @@
+[[install_cnv]]
+=== Installing {ProductName} to {product-title}
+
+The `kubevirt-ansible` RPM contains the latest automation for deploying 
+{ProductName}. This procedure installs all {ProductName} components to your 
+{product-title} cluster. 
+
+This procedure installs the following components:
+
+* {ProductName} core components (KubeVirt)
+* Containerized data importer (CDI) controller
+* Multus, Open vSwitch (OVS), and SR-IOV container network interface plug-ins
+* Updated {product-title} web console
+
+.Prerequisites
+* A running {product-title} 3.11 cluster
+* User with `cluster-admin` privileges
+
+.Procedure
+
+. Log into the {product-title} cluster as an admin user:
++
+----
+$ oc login -u system:admin
+----
+
+. Install the `kubevirt-ansible` RPM and its dependencies:
++
+----
+$ yum install kubevirt-ansible
+----
+
+. If your cluster has unique requirements, customize the 
+`/usr/share/ansible/kubevirt-ansible/vars/cnv.yml` file. Otherwise, skip this 
+step.
+.. Copy the file to your local path:
++
+----
+$ cp /usr/share/ansible/kubevirt-ansible/vars/cnv.yml local-cnv.yml
+----
+.. Edit the copy:
++
+----
+$ vim local-cnv.yml
+----
+
+. Launch {ProductName}:
+** If you use the default `cnv.yml` file, run:
++
+----
+$ ansible-playbook -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml -e 
+@/usr/share/ansible/kubevirt-ansible/vars/cnv.yml 
+/usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml
+----
+** If you copied `cnv.yml` to `~/local-cnv.yml`, run:
++
+----
+$ ansible-playbook -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml -e 
+@~/local-cnv.yml /usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml
+----
+
+. Verify the installation by navigating to the web console at 
+`kubevirt-web-ui.your.app.subdomain.host.com`. Log in by using your 
+{product-title} credentials.
```

Comment:
We'll also want to document deletion of CNV.

```yaml
ansible-playbook -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml -e 
@~/local-cnv.yml /usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml -e apb_action=deprovision
```

Reply:
Thanks, Ryan. A few questions about deprovisioning:

1. Does it matter where the command is run?
2. Are there any prerequisites other than having CNV installed in the first place?
3. What's a good way to test that deprovisioning was successful? Is there output that states the success/failure? If so, please send it so I can use it as an example.

Reply:
@lukas-bednar can probably provide the `oc` command line for checking if there is any kubevirt image running on the cluster.

Reply:
> 1. Does it matter where the command is run?
>
It should be run in the same places as deploy.  That is either in `/usr/share/ansible/kubevirt-ansible` or some place in the user's HOME directory.

> 2. Are there any prerequisites other than having CNV installed in the first place?
>
Having CNV installed is the only prerequisite

> 3. What's a good way to test that deprovisioning was successful? Is there output that states the success/failure? If so, please send it so I can use it as an example.
>
The playbook ran without errors is a success, since it will wait until everything is cleaned up before moving on